tests/conftest.py

```
@@ -25,13 +35,40 @@ def playwright_instance():
 
 @pytest.fixture(scope="session")
 def browser(playwright_instance):
-    browser = playwright_instance.chromium.launch(headless=IS_DOCKER)
+    print(f"\n[Browser Setup] Headless Mode: {HEADLESS_MODE}")
+    # --- 디버깅용 로그 추가 ---
+    print("\n" + "="*30)
+    print(f"Checking Env Vars in CI:")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)